Services/Infrastructure/OJS.Services.Infrastructure/ResilienceStrategies/Implementations/RedisResilienceStrategiesService.cs

```diff
@@ -41,10 +42,7 @@ public async Task<T> ProcessOutcome<T>(Func<Task<T>> fallback, Outcome<T> outcom
 
         if (outcome.Exception is RedisConnectionException or RedisCommandException)
         {
-            if (this.ShouldSendExceptionEmail(outcome.Exception.GetType().Name, outcome.Exception.Message))
```

Comment:
Why is this check removed?

Reply:
The check is present inside the ```SendEmailAsync()``` method.